examples/demo-rollup/tests/evm/evm_logs.rs

```diff
@@ -324,6 +323,28 @@ async fn evm_test_get_logs_at_max_response_size() {
     assert!(iters > 1, "We should have reached the response size limit and been forced to paginate. This test might need adjusting, or pagination is broken.");
 }
 
+#[tokio::test(flavor = "multi_thread")]
+
+async fn evm_test_get_logs_at_max_response_size_without_cursor_throws_error() {
+    let nb_of_txs = 3;
+    let nb_of_logs_per_tx: u32 = 5000;
+    let rollup_and_client = RollupAndClient::new(20_000, EVM_EXTENSION.response_size_limit).await;
```

Comment:
(non-blocking) we can probably speed up this test by reducing the response size limit  and reducing the logs_per_tx value